inference/models/grounding_dino/grounding_dino.py

```diff
@@ -109,12 +115,17 @@ def infer(
         detections = self.model.predict_with_classes(
             image=image,
             classes=text,
-            box_threshold=0.5,
-            text_threshold=0.5,
+            box_threshold=box_threshold,
+            text_threshold=text_threshold,
         )
 
         self.class_names = text
 
+        if CLASS_AGNOSTIC_NMS:
```

Comment:
should this be a request based param like `box_threshold` and `text_threshold`?

Reply:
I agree. Will update this to mimic the YOLO World implementation:
https://github.com/roboflow/inference/blob/2185ed72f094345ee43b39737716a578fd051908/inference/models/yolo_world/yolo_world.py#L89